github/libp2p.yml

```diff
@@ -4804,6 +4804,13 @@ repositories:
     default_branch: main
     delete_branch_on_merge: false
     description: How to use libp2p pubsub in browsers
+    files:
+      .github/pull_request_template.md:
+        content: .github/js_pull_request_template.md
+      .github/workflows/semantic-pull-request.yml:
+        content: .github/workflows/semantic-pull-request.yml
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
```

Comment:
Can these changes be done in a separate pr?

Reply:
Longer explanation of why they appeared here can be found here: https://github.com/libp2p/github-mgmt/pull/198#discussion_r1440325493

Shorter, now that https://github.com/libp2p/github-mgmt/pull/197 has been merged, these changes will disappear from this PR after the rebase.